From 72ee356863aac922bbe3fc4fe16e7dfbf32ae647 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 15 Sep 2009 13:25:21 +0000
Subject: Bug 594320 - Playing youtube videos results in 404 in gst_soup_http_src_parse_status()

The "t" parameter is a bit too escaped (it's escaped in the gdata
xml, and again by urllib.
---
diff --git a/src/plugins/youtube/youtube.py b/src/plugins/youtube/youtube.py
index 1a9046f..11d47ea 100644
--- a/src/plugins/youtube/youtube.py
+++ b/src/plugins/youtube/youtube.py
@@ -334,7 +334,7 @@ class YouTube (totem.Plugin):
 		t_param = self.resolve_t_param (youtube_id)
 
 		if t_param != "":
-			mrl = "http://www.youtube.com/get_video?video_id=" + urllib.quote (youtube_id) + "&t=" + urllib.quote (t_param) + self.get_fmt_string ()
+			mrl = "http://www.youtube.com/get_video?video_id=" + urllib.quote (youtube_id) + "&t=" + t_param + self.get_fmt_string ()
 
 		gobject.idle_add (self._append_to_liststore, treeview_name, pixbuf, entry.title.text, mrl, youtube_id, search_token)
 
--
cgit v0.8.2
